src/common/Voucher.js

```diff
@@ -135,6 +138,7 @@ Voucher.prototype.toArray = function() {
       }
     }
     arr.vouchersourceid = this.vouchersource.id;
+    arr.complaintid = this.complaint.id;
```

Comment:
Will this work when a voucher's created without a complaint?

Reply:
Did you check this works @s-eastwood?

Reply:
I can't remember, I'll double check now....but changes are needed for this once CORE-195 is done.

Reply:
@cdobson I've made a change and believe this is working now